repos/system_upgrade/common/actors/preparepythonworkround/libraries/workaround.py

```diff
@@ -18,6 +19,8 @@ def _get_orig_leapp_path():
 
 def apply_python3_workaround():
     py3_leapp = os.path.join(LEAPP_HOME, 'leapp3')
+    if os.path.exists(LEAPP_HOME):
+        shutil.rmtree(LEAPP_HOME, ignore_errors=True)
     makedirs(LEAPP_HOME)
     leapp_lib_symlink_path = os.path.join(LEAPP_HOME, 'leapp')
     if not os.path.exists(leapp_lib_symlink_path):
```

Comment:
probably we could drop the condition when we are already removing the whole directory in the step before

Reply:
dropped